site/src/pages/components/invokers.explainer.mdx

```diff
@@ -357,6 +357,119 @@ will be `aria-expanded=true` when the _Invokee_ is open and
+security threat considering attackers could already craft a popover.
+
+Modal dialogs also trap focus, such that focus can only be moved within
+elements inside the dialog or the browser native controls (such as the URL
+bar). This is considered to not pose a significant threat as the invoking
+button requires user activation, the dialog cannot render outside of the
+document frame (iframe or browser window) and so cannot occlude the browsers
+Chrome, and there are built in gestures to close a dialog (such as the Esc key
+via a `CloseWatcher`) that cannot be opted out of with or without invokers.
+
+#### Fullscreen elements
+
+Turning an element into a fullscreen needs to be carefully consdidered.
+Scripting already allows this via the `.requestFullscreen()` API which, as the
+name suggests, may not always be successful given the UA context. This API
+requires the document context to be active, for example requiring a user
+activiation (this is true of invokers in the general case), it also requires
+the Permissions Policy to allow for this behaviour, and requires the element to
+not already be on the Top Layer.
+
+All of these constrains should also be true for invokers opening fullscreen
+elements. This means the only _significant_ new behavioural difference between
+an invoker opening a fullscreen element vs the existing behaviour which
+requires scripting, is that sanboxed iframes which disallow scripting but allow
+fullscreen may now declare a button that can fullscreen. Consider the following:
+
+```html
+<iframe sandbox allow="fullscreen">
+  <html id="invokee">
+    <button invoketarget="invokee" invokeaction="toggleFullscreen">Invoke</button> 
+    <button onclick="invokee.requestFullscreen()">Go fullscreen</button>
+  </html>
+</iframe>
+````
+
+In this example, the `<button onclick>` will do nothing as the iframe is
+sanboxed and does not allow scripting. THe `<button invoketarget>` will
+fullscreen the element, however, as the `allow=fullscreen` Permission Policy is
+enabled to allow fullscreen.
+
+This is considered to be a low security threat, as it requires opt-in to the
+fullscreen Permission Policy explitly via iframe attributes or scripting.
+
+#### Media Elements
+
+This proposal allows video and audio elements to be controlled with buttons
+outside of the browsers native video controls. This includes playing, pausing,
+unmuting and invoking the picture-in-picture feature. Today this is only
+possible using scripting via the equivalent APIs: `.play()` / `.pause()` /
+`.muted=` / `.requestPictureInPicture()`. These APIs are guarded by Permissions
+Policy, and so should the invoker equivalent. Some User Agents can be configered
+to reject calls to `.play()`, and this should also be true of invokers. As such
+the key new capability here is the ability to call these APIs without scripting
+enabled. This is effectively the same concern as fullscreen; the security model
+is guarded around the Permissions Policy.
```

Comment:
Not sure on the spec language here but media has the concept of autoplay policy idk if this is permissions policy behind the scenes? It's very possible it is?

Currently I've implemented it with additional user gesture requirements for the "play" action (including through playpause). Depending on the decisions within here this may be superfluous.